docs/docs/widgets/iframe.md

```diff
@@ -2,21 +2,22 @@
 id: iframe
 title: Iframe
 ---
+
 # Iframe
 
-The **Iframe** widget is used to embed another HTML page into the current one and display iframes in your app.
+The **Iframe** component is used to embed another HTML page into the current one and display iframes in your app.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
-| <div style={{ width:"100px"}}> Property </div> |  <div style={{ width:"100px"}}> Description </div> |
-|:--------- |:---------|
-| URL | Set the **URL** of the page to embed.|
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> |
+| :--------------------------------------------- | :------------------------------------------------ |
+| URL                                            | Set the **URL** of the page to embed.             |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Component Specific Actions (CSA)
 
```

Comment:
Hey @dagenius007, I think here on line no. 28 you forgot to remove paddingBottom, please make the change.

Reply:
@RudraPratik30 resolved